particles/Tutorial/Javascript/Demo/Game.arcs

```diff
@@ -0,0 +1,161 @@
+// Tutorial Demo
+// Shows how to use Arcs features to create a tic-tac-toe game.
+import 'GameSchemas.arcs'
+import 'Stores.arcs'
+
+particle Game in './source/Game.js'
+  inout Person {name, avatar, id} playerOne
+  inout Person {name, avatar, id} playerTwo
+  in GameState {gameOver} gameState
+  inout Move {type, move} playerOneMove
+  inout Move {type, move} playerTwoMove
+  inout Move {type, move, playerAvatar} move
+  inout Move {type, move} event
+  check playerOne is private
+  check playerTwo is private
+  consume root
+    provide gameSlot
+
+particle TicTacToeBoard in './source/TicTacToeBoard.js'
+  in GameState {board, currentPlayer} gameState
+  out Move {type, move} event
+  consume boardSlot
+
+particle HumanPlayer in './source/HumanPlayer.js'
+  in GameState {currentPlayer} gameState
+  in Move {type, move} event
+  in Person {id} player
+  out Move {type, move} myMove
+
+particle MoveApplier in './source/MoveApplier.js'
```

Comment:
Why is this a separate particle and not an ability of Game?

Reply:
Combined MoveApplier and Game